packages/radix-ui-themes/src/components/tab-nav.tsx

```diff
@@ -0,0 +1,92 @@
+'use client';
```

Comment:
Do we need 'use client'?

Reply:
Yes we do, because `NavigationMenu` uses a bunch of client stuff (state, context, effects, etc).